cirq/pasqal/pasqal_device.py

```diff
@@ -99,96 +92,56 @@ def validate_operation(self, operation: cirq.ops.Operation):
         Raises:
             ValueError: If the operation is not valid
         """
+
         if not isinstance(operation,
                           (cirq.GateOperation, cirq.ParallelGateOperation)):
             raise ValueError("Unsupported operation")
 
-        if not PasqalDevice.is_pasqal_device_op(operation):
+        if not self.is_pasqal_device_op(operation):
             raise ValueError('{!r} is not a supported '
                              'gate'.format(operation.gate))
 
         for qub in operation.qubits:
-            if not isinstance(qub, ThreeDGridQubit):
-                raise ValueError('{} is not a 3D grid qubit '
+            if not isinstance(qub, self.supported_qubit_type):
+                raise ValueError('{} is not a valid qubit '
```

Comment:
Similar to __init__, please clarify what are the accepted types for this device.